components/Entry/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Link from 'next/link';
+import Skeleton from 'react-loading-skeleton';
+import styles from './Entry.module.css';
```

Comment:
Should we move it to the bottom?

Reply:
Actually, we dont have any code convention at the moment, you should propose it in issue first:
https://github.com/webuild-community/federated-blog/issues/2